files/get-ecr-uri.sh

```diff
@@ -39,15 +59,15 @@ else
     af-south-1)
```

Comment:
Changes looks good to me! Out of curiosity, so does it mean we can actually not maintain this account list anymore but just using one account per partition?

Reply:
No. We should always add a new account when there's a new region, but this means that there's no rush. We can start building the region, and do this at our leisure.